rclpy/src/rclpy_common/src/common.c

```diff
@@ -34,63 +59,53 @@ rclpy_convert_to_py_qos_policy(void * profile)
     return NULL;
   }
 
+  // start qos setting
   rmw_qos_profile_t * qos_profile = (rmw_qos_profile_t *)profile;
+  rclpy_qos_profile_t pyqos;
```

Comment:
nit: Since it is a C struct, I would name this `rclpy_qos` to better distinguish it from the Python object `pyqos_profile`. We typically prefix Python objects with `py`.

Reply:
Yeah good catch